charts/opensearch-cluster/Chart.yaml

```diff
@@ -4,7 +4,7 @@ description: A Helm chart for OpenSearch Cluster
 type: application
 
 ## The opensearch-cluster Helm Chart version
-version: 3.0.0
```

Comment:
Hey @joluc apologies for the wait, can you please revert to `3.0.0` ? 
What I was thinking is along with v2.8.0 https://github.com/opensearch-project/opensearch-k8s-operator/issues/1011 we can do a helm chart release as well. So we can create the version once all changes are merged.
Anyhow without v2.8.0 operator release please correct me if i'm wrong `labels` waont be applied correctly.